package.json

```diff
@@ -127,7 +127,9 @@
     "transform": {
       "^.+\\.js$": "<rootDir>/packages/babel-jest"
     },
-    "setupTestFrameworkScriptFile": "<rootDir>/test_setup_file.js",
```

Comment:
Whoops, this should stay where it was. https://github.com/facebook/jest/issues/4994